dynamodb-stream-reader/src/main/java/elasticsearch/ElasticSearchRestClient.java

```diff
@@ -0,0 +1,124 @@
+package elasticsearch;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import nva.commons.utils.Environment;
+import org.apache.http.HttpHeaders;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+
+import static no.unit.nva.dynamodb.DynamoDBStreamHandler.ELASTICSEARCH_ENDPOINT_INDEX_KEY;
+
+public class ElasticSearchRestClient {
+
+    private static final Logger logger = LoggerFactory.getLogger(ElasticSearchRestClient.class);
+
+    private static final String ELASTICSEARCH_ENDPOINT_OPERATION = "_doc";
+    public static final String INITIAL_LOG_MESSAGE = "using Elasticsearch endpoint {} {} and index {}";
+    public static final String APPLICATION_JSON = "application/json";
+
+    public static final String UPSERTING_LOG_MESSAGE = "Upserting search index  with values {}";
+    public static final String DELETE_LOG_MESSAGE = "Deleting from search API publication with identifier: {}";
+    public static final String ELASTICSEARCH_ENDPOINT_ADDRESS_KEY = "ELASTICSEARCH_ENDPOINT_ADDRESS";
+    public static final String ELASTICSEARCH_ENDPOINT_API_SCHEME_KEY = "ELASTICSEARCH_ENDPOINT_API_SCHEME";
+    public static final String ELASTICSEARCH_ENDPOINT_URI_TEMPLATE = "%s://%s/%s/%s/%s";
```

Comment:
put a comment explaining what each parameter is: e.g. `https://host/indexName/folderName/file`

Reply:
added clarifying comment